packages/wonder-blocks-dropdown/components/action-menu.md

```diff
@@ -187,3 +187,82 @@ const styles = StyleSheet.create({
     </ActionMenu>
 </View>
 ```
+
+### Example: Opening an ActionMenu programmatically
+
+Sometimes you'll want to trigger a dropdown programmatically. This can be done by
+setting the `opened` prop to `true`. In this situation the `ActionMenu` is a
```

Comment:
It can also be the case that `false` works, too - maybe be good to make clear in this paragraph that any boolean passed in to `opened` will cause it to be controlled. (I missed this the first time I saw this!)